epsagon/trace.py

```diff
@@ -953,6 +953,15 @@ def remove_ignored_keys(self, input_dict):
                 value = input_dict[key]
                 if isinstance(value, dict):
                     copied_dict[key] = self.remove_ignored_keys(value)
+                else:
+                    try:
+                        json_value = json.loads(value)
```

Comment:
Consider checking if value is str before calling json.loads

Reply:
Right 👍